README.md

```diff
@@ -0,0 +1,642 @@
+Puppetboard
+===========
+
+[![image](https://travis-ci.org/voxpupuli/puppetboard.svg?branch=master)](https://travis-ci.org/voxpupuli/puppetboard)
+
+[![image](https://coveralls.io/repos/github/voxpupuli/puppetboard/badge.svg?branch=master)](https://coveralls.io/github/voxpupuli/puppetboard?branch=master)
+
+Puppetboard is a web interface to [PuppetDB](http://docs.puppetlabs.com/puppetdb/latest/index.html) aiming to replace the reporting functionality of [Puppet Dashboard](http://docs.puppetlabs.com/dashboard/).
+
+Puppetboard relies on the [pypuppetdb](https://pypi.python.org/pypi/pypuppetdb) library to fetch data from PuppetDB and is built with the help of the [Flask](http://flask.pocoo.org) microframework.
+
+As of version 0.1.0 and higher, Puppetboard **requires** PuppetDB 3. Version 0.3.0 has been tested with PuppetDB versions 3 through 5.
+
+At the current time of writing, Puppetboard supports the following Python versions:
+
+ * Python 2.6
+ * Python 2.7
+ * Python 3.6
+
+![View of a node](screenshots/overview.png)
+
+Installation
+------------
+
+Puppetboard is now packaged and available on PyPi.
+
+### Production
+
+#### Puppet module
+
+There is a [Puppet module](https://forge.puppetlabs.com/puppet/puppetboard) by [Spencer Krum](https://github.com/nibalizer) that takes care of installing Puppetboard for you.
+
+You can install it with:
+
+> puppet module install puppet-puppetboard
+
+To see how to get it working with EL7 check out these [docs](https://github.com/voxpupuli/puppetboard/blob/master/docs/EL7.md).
+
+#### Manual
+
+To install it simply issue the following command:
+
+``` {.sourceCode .bash}
+$ pip install puppetboard
+```
+
+This will install Puppetboard and take care of the dependencies. If you do this Puppetboard will be installed in the so called site-packages or dist-packages of your Python distribution.
+
+The complete path on Debian and Ubuntu systems would be `/usr/local/lib/pythonX.Y/lib/dist-packages/puppetboard` and on Fedora would be `/usr/lib/pythonX.Y/site-packages/puppetboard`
+
+where X and Y are replaced by your major and minor python versions.
+
+You will need this path in order to configure your HTTPD and WSGI-capable application server.
+
+#### Packages
+
+Native packages for your operating system will be provided in the near future.
+
+<table>
```

Comment:
Shouldn't this be a native markdown table? -> https://help.github.com/articles/organizing-information-with-tables/

Reply:
That would be a great contribution! This PR is simply meant to get us to markdown in the simplest way possible so further contributions can be done easily. The motivation is from spending entirely too much time trying to figure out RST where that took longer than the fix I was implementing.  The conversion is done using pandoc. https://pandoc.org/